mv/src/mv.yml

```diff
@@ -0,0 +1,71 @@
+name: mv
+version: "0.1"
+author: Gab David <g.bianchetdavid@gmail.com>
+about: Move (rename) files
```

Comment:
I think it would be better if it was "Move or rename files"

Reply:
I just copied the GNU version--just changed it.